mojo/core/entrypoints.cc

```diff
@@ -354,7 +354,7 @@ MojoResult MojoSendInvitationImpl(
 #endif
 }
 #if defined(CASTANETS)
-MojoResult MojoRetryInvitation(
+MojoResult MojoRetryInvitationImpl(
```

Comment:
Please check [line 438](https://github.com/Samsung/Castanets/pull/233/files#diff-a5982dce8dad2b629215942bd6a1a54aR438)

Reply:
Thank you for your review:) Done.